ui/perfherder/alerts/AlertsView.jsx

```diff
@@ -313,8 +313,14 @@ class AlertsView extends React.Component {
         <Container fluid className="pt-5 max-width-default">
           {loading && <LoadingSpinner />}
 
+          {perfErrorMessages.length > 0 && (
+            <Container className="px-0 max-width-default">
+              <ErrorMessages errorMessages={perfErrorMessages} />
+            </Container>
+          )}
+
```

Comment:
There is no need to have a second `ErrorMessages` component added. You should set the `errorMessages` state default value to `perfErrorMessages` if true, otherwise default to []. Also, there is no need to name it `perfErrorMessages` - simply 'errorMessages' is fine.

Reply:
So basically this AlertsView component will initialize with any errorMessages passed as a prop from the parent component (App).